src/components/CoinPageChart/CoinPageChart.js

```diff
@@ -71,27 +104,23 @@ export function CoinPageChart(props) {
             value={168}
             name="7d"
           />
-          <div></div>7d
+          <div></div><p>7d</p>
```

Comment:
why do you have a div an a p?

Reply:
The div is for the inner circle of the radio button when clicked